ethclient/gethclient/gethclient_test.go

```diff
@@ -81,9 +81,12 @@ func generateTestChain() (*core.Genesis, []*types.Block) {
 			rcfg.L1GasPriceOracleAddress: {
 				Balance: big.NewInt(0),
 				Storage: map[common.Hash]common.Hash{
-					rcfg.L1BaseFeeSlot: common.BigToHash(big.NewInt(10000)),
-					rcfg.OverheadSlot:  common.BigToHash(big.NewInt(10000)),
-					rcfg.ScalarSlot:    common.BigToHash(big.NewInt(10000)),
+					rcfg.L1BaseFeeSlot:     common.BigToHash(big.NewInt(10000)),
+					rcfg.OverheadSlot:      common.BigToHash(big.NewInt(10000)),
+					rcfg.ScalarSlot:        common.BigToHash(big.NewInt(10000)),
+					rcfg.L1BlobBaseFeeSlot: common.BigToHash(big.NewInt(10000)),
+					rcfg.CommitScalarSlot:  common.BigToHash(big.NewInt(10000)),
+					rcfg.BlobScalarSlot:    common.BigToHash(big.NewInt(10000)),
```

Comment:
probably not relevant but IsCurie is false here. We might need to check if config has Curie enabled on genesis?

Reply:
Yeah, not relevant here, but added in [ec5c347](https://github.com/scroll-tech/go-ethereum/pull/755/commits/ec5c3478399b15307561b7af0ea6f103f2ec496a) for completeness' sake.